tools/debian/control

```diff
@@ -0,0 +1,116 @@
+Source: cockpit
+Section: admin
+Priority: optional
+Maintainer: Michael Biebl <biebl@debian.org>
+Build-Depends: debhelper (>= 9),
+               dh-autoreconf,
+               autoconf,
+               automake,
+               intltool,
+               libssh-dev,
+               libssl-dev,
+               zlib1g-dev,
+               libkrb5-dev,
+               libxslt1-dev,
+               libkeyutils-dev,
+               libglib2.0-dev,
+               libsystemd-dev,
+               libpolkit-agent-1-dev,
+               libpcp3-dev,
+               libjson-glib-dev,
+               libpam0g-dev,
+               libpcp-import1-dev,
+               libpcp-pmda3-dev,
+               xsltproc,
+               xmlto,
+               docbook-xsl,
+               glib-networking,
+               nodejs-legacy,
+               npm,
+               openssh-client
+Standards-Version: 3.9.6
+Homepage: http://cockpit-project.org/
+#Vcs-Git: git://anonscm.debian.org/collab-maint/cockpit.git
+#Vcs-Browser: https://anonscm.debian.org/cgit/collab-main/cockpit.git
+
+Package: cockpit
+Architecture: any
+Depends: ${misc:Depends},
+         cockpit-bridge,
+         cockpit-ws,
+         cockpit-shell,
+         xdg-utils
+Recommends: cockpit-networkmanager,
+            cockpit-docker
+Suggests: cockpit-doc,
+          cockpit-pcp
+Description: User interface for Linux servers
```

Comment:
The Debian package split as found at https://anonscm.debian.org/cgit/collab-maint/cockpit.git/ is indeed largely based on what's in tools/cockpit.spec, including package descriptions and list of files (although some paths differ on Debian, e.g. because of $libexecdir)